models/issues/review.go

```diff
@@ -295,15 +301,37 @@ func CreateReview(ctx context.Context, opts CreateReviewOptions) (*Review, error
 		CommitID:     opts.CommitID,
 		Stale:        opts.Stale,
 	}
-	if opts.Reviewer != nil {
+
+	switch {
+	case opts.Reviewer != nil:
```

Comment:
Is this the new `if else`?

Reply:
yes ... compile whise it does no difference ... but it is more redble?

and most linter enforce itbif tjere would be one more ifelse

Reply:
So there is no reason to not use `if else` here.

Reply:
well looks that is prefered here i'll change it back